src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/brev/domene/BrevPeriodeTestKlasser.kt

```diff
@@ -66,6 +66,7 @@ data class BrevPeriodeTestConfig(
 data class BrevPeriodeTestKompetanse(
     val id: String,
     val søkersAktivitet: SøkersAktivitet,
+    val søkersAktivitetsLand: String?,
```

Comment:
```suggestion
    val søkersAktivitetsland: String?,
```
Ser at vi bruker liten l andre steder, så vi bør kjøre på med det her og.

Reply:
Fiksa i https://github.com/navikt/familie-ba-sak/pull/2799/commits/2f61ee4e0a9069a6749394381ca737eb8010ceb0